soundbay/data.py

```diff
@@ -232,6 +217,46 @@ def __len__(self):
         return self.metadata.shape[0]
 
 
+
+
+class MultiClassifierDataset(ClassifierDataset):
```

Comment:
I think the naming is quite confusing. Why call is `MultiClassifierDataset` and `ClassifierDataset`, if the `ClassifierDataset` doesn't support more than two classes?
Or does it?

Reply:
I would suggest a way for solving this issue, if I may:
having a base class dataset that is called `BaseDataset` with all the methods implemented except of `_get_audio` (that will raise NotImplementedError exception)
We'll have two classes to use:
`ClassifierDataset` - a dataset to be used with a background label, make sure it's label 0. The current implementation is actually the same, only we have the bug @noamzbr stated with taking margin from other call. We'll need to have some logic to make sure we only take from bg. As far as I'm concerned, we can keep the current one with a technical debt and implement it in the future, so we could merge the current PR asap.
`NoBackgroundClassifierDataset` - The same but this time there is no margin at all, just sampling from the calls. This dataset is gonna be used for training on Liz's data